### ISSUE TITLE
Checksumming is slow, not using available CPU

### ISSUE THREAD

**Issue by magnush0lm** [jotta/jotta-cli-issues#141]
Make sure you are running the latest version of jotta-cli before reporting an issue.

**jotta-cli release (`jotta-cli version`):**
jottad version    : 0.9.41303
jotta-cli version : 0.9.41303

**Description of problem:**
Checksumming files take much more time than expected

**Expected:**
Checksumming should (be able to) use all available CPU power to do the calculations.
A couple of tests show that checksumming speed averages at ~30MiB/s on my computer, the CPU is comfortable below 15% utilization.

Upload speed, however is range from about 30MiB/s up to 80MiB/s.

One would think that calculating the checksum of a file would be faster than uploading it to Jottacloud.

**jotta-cli status (`jotta-cli status`):**
```

```

**Relevant logs for the issue (`~/.jottad/jottabackup.log or /var/lib/jotta/jottabackup.log ` )**
```

```

**Traceback**
```

```

**Additional info:**

**Comment by coopersimon** [comment]
You can try `jotta-cli config set checksumreadrate [rate in B/s]`. The default is 50MiB/s.

**Comment by magnush0lm** [comment]
> You can try `jotta-cli config set checksumreadrate [rate in B/s]`. The default is 50MiB/s.

Thanks!
Should this config value be shown when you do ```jotta-cli config get``` before setting it?
It looks like the other options output their default values:
```
jotta-cli.exe config get
downloadrate          : unlimited
uploadrate            : unlimited
maxdownloads          : 2
maxuploads            : 2
scaninterval          : 1h0m0s
webhookstatusinterval : 6h0m0s
logscanignores        : false
slowmomode            : 0
logtransfers          : false
photosmonthastext     : true
OK
```

**Comment by magnush0lm** [comment]
Also, can you set ```checksumreadrate``` to "unlimited" ?

**Comment by coopersimon** [comment]
`checksumreadrate` is a hidden config option so it won't show up in `config get`. We are planning on showing it in the next release. We also identified a bug with it that will be fixed in the next release - the option isn't saved correctly and needs to be re-set each time you start `jottad`.

You can set it to unlimited.

**Comment by coopersimon** [comment]
We released 0.10.42740 which exposes and fixes this option.

**Comment by michalsykora** [comment]
to set unlimited checksum read rate:
`jotta-cli config set checksumreadrate 0`
